streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -4821,8 +4911,10 @@ private Map<TaskId, StateMachineTask> handleAssignment(final Map<TaskId, Set<Top
         when(standbyTaskCreator.createTasks(standbyAssignment)).thenReturn(standbyTasks);
         when(activeTaskCreator.createTasks(any(), Mockito.eq(allActiveTasksAssignment))).thenReturn(allActiveTasks);
 
-        expectRestoreToBeCompleted(consumer);
-        replay(consumer);
+        final Set<TopicPartition> assignment = singleton(new TopicPartition("assignment", 0));
+        lenient().when(mockitoConsumer.assignment()).thenReturn(assignment);
```

Comment:
`lenient()` was needed here because this method is a dependency on quite a few tests some of which do not actually call this. The tests which do not call it are
* `shouldClassifyExistingTasksWithoutStateUpdater`
* `shouldComputeOffsetSumForNonRunningActiveTask`
* `shouldSkipUnknownOffsetsWhenComputingOffsetSum`
* `shouldReportLatestOffsetAsOffsetSumForRunningTask`